article/app/model/dotcomponents/DotcomponentsDataModel.scala

```diff
@@ -1,31 +1,25 @@
 package model.dotcomponents
 
-import ai.x.play.json.Jsonx
 import com.gu.contentapi.client.model.v1.ElementType.Text
 import com.gu.contentapi.client.model.v1.{BlockElement => ClientBlockElement, Blocks => APIBlocks}
 import common.Edition
 import common.Maps.RichMap
 import common.commercial.{CommercialProperties, EditionCommercialProperties, PrebidIndexSite}
-import conf.{Configuration, Static}
 import conf.Configuration.affiliatelinks
 import conf.switches.Switches
+import conf.{Configuration, Static}
 import controllers.ArticlePage
 import model.SubMetaLinks
 import model.content.Atom
 import model.dotcomrendering.pageElements.{DisclaimerBlockElement, PageElement}
-import model.meta._
+import model.meta.{LinkedData, PotentialAction, _}
```

Comment:
This was Intellij but on closer inspection it doesn't make sense at all! Will fix.